### ISSUE TITLE
Action blocked keeps appearing

### ISSUE THREAD

**Issue by lenoirrichelieu** [teelolws/EditModeExpanded#58]
I still have issues with action blocked by addon. When in combat pressing a keybind gives the error. Here is the bug details:

3x [ADDON_ACTION_BLOCKED] AddOn 'EditModeExpanded' tried to call the protected function 'UIParentRightManagedFrameContainer:SetPoint()'.
[string "@!BugGrabber/BugGrabber.lua"]:480: in function <!BugGrabber/BugGrabber.lua:480>
[string "=[C]"]: in function `SetPoint'
[string "@SharedXML/AnchorUtil.lua"]:34: in function `SetPoint'
[string "@FrameXML/UIParent.lua"]:3095: in function `UIParentManageFramePositions'
[string "@FrameXML/UIParent.lua"]:2570: in function <FrameXML/UIParent.lua:2557>
[string "=[C]"]: in function `SetAttribute'
[string "@FrameXML/UIParent.lua"]:3113: in function `UIParent_ManageFramePositions'
[string "@FrameXML/EditModeManager.lua"]:640: in function `UpdateBottomActionBarPositions'
[string "@FrameXML/EditModeManager.lua"]:554: in function `UpdateActionBarLayout'
[string "@FrameXML/EditModeSystemTemplates.lua"]:655: in function `UpdateGridLayout'
[string "@FrameXML/ActionButton.lua"]:368: in function `UpdateAction'
[string "@FrameXML/ActionButton.lua"]:795: in function `OnEvent'
[string "@FrameXML/ActionButton.lua"]:216: in function <FrameXML/ActionButton.lua:213>

After relog everything is fine but if in combat is quite annoying to reload or to click buttons until fight is over. I have the latest version and is the only addon related to edit mode or action bars.

**Comment by teelolws** [comment]
I get this sometimes too. But so far its always random when it happens. Until I find a consistent way to always make the error happen, I can't begin to fix it.

**Comment by teelolws** [comment]
I haven't had this lately, presume Blizzard fixed it on their end, or it was another addon causing it. If it happens again, please make a new issue with the new error message.
